openedx/core/djangoapps/models/course_details.py

```diff
@@ -43,6 +47,10 @@ def __init__(self, org, course_id, run):
         self.enrollment_start = None
         self.enrollment_end = None
         self.syllabus = None  # a pdf file asset
+        self.title = ""
```

Comment:
As I understand it, these about fields end up getting persisted as standalone modules(xblocks) in mongo and are referenced by the course_module. Eventually, the data stored in these fields will get displayed on a redesigned course details page.